src/pj_obs_api.c

```diff
@@ -384,14 +384,23 @@ PJ  *proj_create_crs_to_crs (PJ_CONTEXT *ctx, const char *srid_from, const char
     function can be extended to support "late-binding" transformations in the
     future without affecting users of the function.
 
+    An "area of use" can be specified in area. In the current version of this
+    function is has no function, but is added in anticipation of a
+    "late-binding" implementation in the future. The idea being, that if a user
+    supplies an area of use, the correct transformatin between to given systems
```

Comment:
"the correct transformation" (typo: missing o at end of transformation): there might be several candidates. So I'd say "a more accurate transformation"